src/basic/fileio.c

```diff
@@ -363,32 +363,38 @@ int verify_file(const char *fn, const char *blob, bool accept_extra_nl) {
         return 1;
 }
 
-int read_full_virtual_file(const char *filename, char **ret_contents, size_t *ret_size) {
+int read_virtual_file(const char *filename, size_t max_size, char **ret_contents, size_t *ret_size) {
```

Comment:
wouldn't read_full_virtual_file_full() be  a better name?

Reply:
The patch makes it *not* read the full file… So I think it's fine to call it `read_virtual_file` without further details in the name.